cmd/bad-key-revoker/main.go

```diff
@@ -370,9 +370,11 @@ func (bkr *badKeyRevoker) invoke(ctx context.Context) (bool, error) {
 		}
 	}
 
-	revokerEmails := idToEmails[unchecked.RevokedBy]
-	bkr.logger.AuditInfo(fmt.Sprintf("revoking certs. revoked emails=%v, emailsToCerts=%s",
-		revokerEmails, emailsToCerts))
+	serials := make([]string, 0)
```

Comment:
```suggestion
	var serials []string
```

This is more in keeping with local style (and what I think is overall good go style).